src/backend/PortabilityService.Functions/Targets.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PortabilityService.Functions
+{
+    public static class Targets
+    {
+        /// <summary>
+        /// Returns the list of supported targets 
+        /// </summary>
+        /// <param name="req">Inbound request for targets</param>
+        /// <param name="log">logging provider instance</param>
+        /// <returns></returns>
+        [FunctionName("targets")]
+        public static async Task<HttpResponseMessage> Run(
+                    [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "targets")] HttpRequestMessage req, ILogger log)
+        {
+            var targets = await GetStaticListOfTargets(log);
+            var content = new StringContent(targets, Encoding.UTF8, "application/json");
+            return new HttpResponseMessage { StatusCode = System.Net.HttpStatusCode.OK, Content = content};
+        }
+
+        /// <summary>
+        /// This method is a placeholder, and will be replaced with an asynchronous call to fetch catalog data in a future sprint
+        /// </summary>
+        /// <returns>A Task<string> representing the async task that fetches the catalog of supported targets</string></returns>
+        /// <param name="log">logging provider instance</param>
+        private static async Task<string> GetStaticListOfTargets(ILogger log)
```

Comment:
You can remove `async/await` and just return the Task. This removes the overhead of creating that entire async/await state machine if you're not doing anything with the Task after it completes inside this function.